CHANGES.rst

```diff
@@ -12,6 +12,9 @@ Unreleased
   unmarking a submission as original content.
 * Parameter ``without_websockets`` to :meth:`~.Subreddit.submit_image` and
   :meth:`~.Subreddit.submit_video` to submit without using WebSockets.
+* :meth:`.Reddit.redditor` supports fullname param to fetch a Redditor
```

Comment:
This is a nitpick, but can you please surround "fullname" on this line with double backticks like:

    ... supports ``fullname`` param ...

The second instance of the word "fullname" in this sentence is fine as is, but I want the first one surrounded by backticks so that it looks like a parameter.